code/game/objects/structures/lavaland/geyser.dm

```diff
@@ -65,7 +65,7 @@
 
 /obj/item/plunger/reinforced
 	name = "reinforced plunger"
-	desc = " It's an M. 7 Reinforced Plunger� for heavy duty plunging."
+	desc = " It's an M. 7 Reinforced Plunger� for heavy duty plunging."
```

Comment:
.

Reply:
did i change this what

Reply:
still here :(

Reply:
im confused i never touched thisd file

Reply:
wat do

Reply:
how the fuck did this even happen.. apparently it was related to the mass replace i did

Reply:
you can prob webedit it back in